ci-operator/config/maistra/istio-images-centos/maistra-istio-images-centos-maistra-2.4.yaml

```diff
@@ -29,6 +29,58 @@ tests:
     make lint
   container:
     from: src-test
+- as: istio-images-centos-daily-images-2.4
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 4h0m0s
+    version: "4.12"
+  cron: 00 03 * * *
+  steps:
+    env:
+      MAISTRA_BUILDER_IMAGE: registry.ci.openshift.org/ci/maistra-builder:2.4
+      MAISTRA_MC_POD: not-used
+      MAISTRA_NAMESPACE: maistra-daily-images
+      MAISTRA_SC_POD: maistra-daily-images-pod
+    test:
+    - as: copy-src
+      cli: latest
+      commands: |
+        # copy the script to privileged pod
+        oc cp create-images.sh "${MAISTRA_NAMESPACE}"/"${MAISTRA_SC_POD}":/work/
+      from: src-test
+      resources:
```

Comment:
You need something similar to 
https://github.com/openshift/release/blob/master/ci-operator/config/maistra/istio/maistra-istio-maistra-2.3.yaml#L158-L160

In order to access the env defined in steps, you have to mention env variable names in each test section.